website/content/docs/agent/options.mdx

```diff
@@ -1267,6 +1267,11 @@ bind_addr = "{{ GetPrivateInterfaces | include \"network\" \"10.0.0.0/8\" | attr
       for more than twice the _current_ `leaf_cert_ttl`, it will be removed
       from the trusted list.
 
+    - `root_cert_ttl` ((#todo)) todo. Defaults to 10 years as `87600h`.
+      todo restrictions.
```

Comment:
Still needs a description (leftover todo)

Reply:
By restrictions I guess you mean "is only used when the CA system is first initialized, or when a configuration change causes a rotation or the root certificate" ? I think we should say something about that.

Reply:
I was punting on the docs a bit to make sure that the stuff here was something we eventually wanted to check in.

Which looks like it is case. So I added the docs now :)

Reply:
@FFMMM Can you also make sure this field gets documented under https://github.com/hashicorp/consul/blob/main/website/content/partials/http_api_connect_ca_common_options.mdx?

The options in this file are displayed under the provider-specific CA config options. For example, https://www.consul.io/docs/connect/ca/vault#common-ca-config-options.